### DISCUSSION TITLE
Skins on lazer

### DISCUSSION THREAD

**Discussion by MGWpache** [ppy/osu#23169] [Q&A] [answered]
Hii, is there any possibility to import my osu! stable skins to my osu! lazer?, default skins aren't that bad but i prefer mines hahaha.

**Comment by hazelcode** [comment]
Yes! let me explain

**Reply by hazelcode** [reply, marked as answer]
> Yes! let me explain

Step 1. Click Settings / Ctrl + O

Step 2. Click "Run setup wizard"
![osu_2023-04-08_20-24-10](https://user-images.githubusercontent.com/84425557/230750960-1e620bfa-fe66-4f58-9cc1-54566d408bc0.jpg)

Step 3. Skip to the import screen, select "Skins" option and click "Import content from previous version"
![osu_2023-04-08_20-24-26](https://user-images.githubusercontent.com/84425557/230751005-4ab8847f-493a-40b5-918c-50d05b0d0a3c.jpg)
